### ISSUE TITLE
components is empty

### ISSUE THREAD

**Issue by a44281071** [dapr/installer-bundle#12]
OS: windows 10
docker: windows docker-desktop
dapr: daprbundle_windows_amd64.zip

Result of `init --from-dir`:
- .dapr/componets folder is empty.
- no dapr_zipkin docker container.
- no dapr_redis docker container.

**Comment by shivamkm07** [comment]
Hi @a44281071, this is expected behavior. Currently we don't bundle zipkin/redis containers in daprbundle to avoid dependency on these third party packages. Since the containers ain't there, corresponding component yaml files are also removed from `.dapr/components` folder. 

For downloading zipkin/redis, you can pull them manually:
```
1. docker run --name "dapr_zipkin" --restart always -d -p 9411:9411 openzipkin/zipkin
2. docker run --name "dapr_redis" --restart always -d -p 6379:6379 redislabs/rejson
```
And the component files need to be created manually as well.

**Comment by shivamkm07** [comment]
> 

It does require an update in the README describing the same, will do.

**Comment by shivamkm07** [comment]
Closed by #18
